component/core/src/main/java/io/meeds/social/core/upgrade/SpaceNavigationIconUpgradePlugin.java

```diff
@@ -77,14 +80,23 @@ public boolean shouldProceedToUpgrade(String newVersion, String previousVersion)
     return !spaceNodes.isEmpty();
   }
 
+  @Override
+  public boolean shouldProceedToUpgrade(String newVersion,
+                                        String previousGroupVersion,
+                                        UpgradePluginExecutionContext previousUpgradePluginExecution) {
+
+    int executionCount = previousUpgradePluginExecution == null ? 0 : previousUpgradePluginExecution.getExecutionCount();
+    return !isExecuteOnlyOnce() || executionCount == 0;
+  }
```

Comment:
I think that this is useless since the UP is renamed, it will be executed for the first time

Reply:
done !